src/radium-stylesheet.js

```diff
@@ -0,0 +1,326 @@
+// -------------------------------
+// RADIUM THEME
+// -------------------------------
+// theme
+import theme from "./radium-theme";
+
+// For both akkurat & sharp sans no2 bold:
+// MozOsxFontSmoothing: "grayscale",
+// WebkitFontSmoothing: "antialiased"
+
+const codeStyles = {
+  fontFamily: theme.code,
+  fontSize: ".9em",
+  lineHeight: 1.375,
+  direction: "ltr",
+  textAlign: "left",
+  whiteSpace: "pre",
+  wordSpacing: "normal",
+  wordBreak: "normal",
+  tabSize: 2,
+  webkitHyphens: "none",
+  mozHyphens: "none",
+  msHyphens: "none",
+  hyphens: "none",
+  background: "transparent"
+};
+
+// Stylesheet
+export default {
+  "*, *:before, *:after": {
+    WebkitBoxSizing: "inherit",
+    MozBoxSizing: "inherit",
```

Comment:
`BoxSizing` is prefixed here but not for `body: {}`. I forgot what's happening here. Is autoprefixer missing?